MODULE.bazel

```diff
@@ -17,3 +20,33 @@ archive_override(
     strip_prefix = "gz-utils-gz-utils3",
     urls = ["https://github.com/gazebosim/gz-utils/archive/refs/heads/gz-utils3.tar.gz"],
 )
+
+PYTHON_VERSIONS = [
+    "3.9",
+    "3.10",
+    "3.11",
+    "3.12",
+]
+
+python = use_extension("@rules_python//python/extensions:python.bzl", "python")
+
+[
+    python.toolchain(
+        is_default = python_version == PYTHON_VERSIONS[-1],
+        python_version = python_version,
+    )
+    for python_version in PYTHON_VERSIONS
+]
+
+pip = use_extension("@rules_python//python/extensions:pip.bzl", "pip")
+
+[
+    pip.parse(
+        hub_name = "xacro_python_dependencies",
+        python_version = python_version,
+        requirements_lock = "//bazel:requirements_lock.txt",
```

Comment:
It looks like `//bazel:requirements_lock.txt` is missing from this PR

Reply:
Removed.